index.html

```diff
@@ -255,29 +287,44 @@ <h2 class="ui header">Separate glyphs</h2>
         console.log(window[name + "Glyphs"]);
         */
 
-    $('.fontname').text(window['name'] + ' — ' + window['size'] + 'pt').show()
+    $('.fontname').text(window['name']).show()
+
+    const glyphsArray = window[name + 'Glyphs']
+
+    // Calculate the max yAdv to scale the table columns for the glyphs
+    let maxYAdv = 0
```

Comment:
This per-glyph number at index [3] is xadvance, not yadvance
There's only one yadvance per font.

Reply:
Yes it's `xadvance` — just a naming issue you're right. This sizing I'm doing here is only very indicative and is only used for the table of glyphs. I don't think there is a "right" way of doing it, it's simply a hack to avoid having too many glyphs per row when the font is "big"